sls-versions/src/main/java/com/palantir/sls/versions/SlsVersionMatcherParser.java

```diff
@@ -121,7 +121,7 @@ private static long number(String string, int startIndex) {
             try {
                 return ok(next, Integer.parseUnsignedInt(string.substring(startIndex, next)));
             } catch (NumberFormatException e) {
-                if (e.getMessage().endsWith("exceeds range of unsigned int.")) {
+                if (e.getMessage() != null && e.getMessage().endsWith("exceeds range of unsigned int.")) {
```

Comment:
Might as well simplify things for the compiler:
```suggestion
                String message = e.getMessage();
                if (message != null && message.endsWith("exceeds range of unsigned int.")) {
```